### ISSUE TITLE
Feature: Acceptance convert to Litmus

### ISSUE THREAD

**Issue by cdenneen** [dylanratcliffe/onceover#311]
With the deprecation of **beaker** the `onceover run acceptance` should be converted over to using litmus.

**Comment by cdenneen** [comment]
Similar to module testing after you run `litmus:provision_list` and `litmus:install_agent` rather than `litmus:install_module` is where you'd basically run the `Onceover::Deploy.new.deploy_local(repo)` and then use `LitmusHelper.instance.bolt_upload_file` to copy the files over. 

It might be better to create new function for `deploy_remote(repo)` that copies the `Puppetfile` to the target host and then runs the `r10k puppetfile install` (`LitmusHelper.instance.run_shell`) (**remember** have to account for `site`,`site-modules`,`dist`, ***whatever you use*** to be copied over as well.

```ruby
desc 'Onceover Deploy'
task :onceover_deploy do
  require 'onceover/controlrepo'
  require 'onceover/deploy'
  repo = Onceover::Controlrepo.new
  Onceover::Deploy.new.deploy_local(repo)
end
```

**Comment by dylanratcliffe** [comment]
Yeah so a lot of work went into the `acceptance` branch that did use Litmus but the work stalled after I hit some issues with Bolt (they have since been resolved) and I got moved to a management position and had less time hands-on-keyboard. It was pretty close and I would love to get it finished at some point: https://github.com/dylanratcliffe/onceover/tree/acceptance

**Comment by cdenneen** [comment]
@dylanratcliffe anyway you can rebase that brand so it's not so far behind main? I'd like to give it a test.
